### PR TITLE
Improve GitHub templates.

### LINKED ISSUE #1336: Improves Github TEMPLATE
The Github template is not good enough so I want to improve it. Moved from https://github.com/vscode-icons/vscode-icons/pull/1335.

### PR DESCRIPTION
**Pull request by JimiC**
This PR is inspired by @torn4dom4n and should fix #1336 and close #1335.

If you think that these proposed changes can be improved further or you believe that they require more changes, please feel free to discuss them here.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
+1. Do a [search](https://github.com/vscode-icons/vscode-icons/issues?q=is%3Aopen+is%3Aissue) before opening a new issue. If it exists, add a comment to it instead of opening a new one.
+1. Fill in the provided issue template.
 1. Include screenshots or GIFs whenever possible.
 1. Don't be rude, describe your problem cool.
 
 **Pull Request:**
 
-1. Use the present tense.
+1. Use the present tense in your commit message.
+1. Fill in the provided pull request template**.
```
Comment (jens1o) [inline .github/CONTRIBUTING.md]: what does `**` at the end refer to?

Comment (JimiC) [reply .github/CONTRIBUTING.md]: Copy-paste leftover. Nice catch.

**Comment by torn4dom4n** [comment]
@JimiC I think you should remove example comment in issues templstes to contributing. Do you think that?

**Comment by JimiC** [comment]
@torn4dom4n My opinion is that all this info inside the templates should be moved to the wiki and templates should only contain the scaffold plus the link to the wiki page.

**Comment by torn4dom4n** [comment]
It is a good idea!

**Comment by robertohuertasm** [comment]
I agree with you @JimiC :+1:

**Comment by robertohuertasm** [comment]
Are you willing to make the move before next release? Although, this is not affecting the deliverable so... no hurries!

**Comment by JimiC** [comment]
Wiki pages have been added.

File: README.md [hunk 2]
```diff
@@ -81,6 +81,8 @@ For those using [Docker](https://www.docker.com/) checkout the [docker section](
 
 On the other hand, if you want to show the world your artistic side, checkout the [contributing section](https://github.com/vscode-icons/vscode-icons/wiki/Contributing).
 
+On any case, please follow our [contribution guidelines](https://github.com/vscode-icons/vscode-icons/blob/master/.github/CONTRIBUTING.md).
```
Comment (jens1o) [inline README.md]: `In any case`?

**Pull request merged**